php/containers.json

```diff
@@ -201,6 +201,7 @@
         "TALK_RECORDING_HOST=nextcloud-aio-talk-recording"
       ],
       "restart": "unless-stopped",
+      "stop_grace_period": 600,
```

Comment:
@Zoey2936 do you think 10min are enought to wait for cronjobs to stop?

Reply:
@Zoey2936 if wait only waits for the sleep command, 5min should be enough?

Reply:
no

Reply:
https://docs.docker.com/compose/compose-file/compose-file-v3/#stop_grace_period

Reply:
and why is this needed here?